readthedocs.yml

```diff
@@ -7,7 +7,7 @@ build:
   image: latest
 
 python:
-  version: 3.7
+  version: 3.10
```

Comment:
It doesn't look it likes this version 
<img width="771" alt="Screenshot 2023-02-06 at 14 06 29" src="https://user-images.githubusercontent.com/7579677/216967326-7096d912-8bff-4f51-951b-39d8509dd9ff.png">

Reply:
Yeah, I saw that. Decided there was more to bringing it back to life, so working on that in #144